templates/base.html

```diff
@@ -28,8 +76,7 @@
     </div>
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Did you experiment with removing jquery and seeing what happened? I wonder if it can be yanked at this point.

Reply:
Not yet, will try and report back